src/xrCore/xr_ini.h

```diff
@@ -22,6 +22,13 @@ class IReader;
 class XRCORE_API CInifile
 {
 public:
+    enum
+    {
+        eSaveAtEnd = 1 << 0,
+        eReadOnly = 1 << 1,
+        eOverrideNames = 1 << 2,
+    };
+    Flags8 m_flags;
```

Comment:
Don't make it public. It's ok to make enum public, but not the variable.

Reply:
Fixes